public/templates/teacher_sets/detail.html

```diff
@@ -6,8 +6,11 @@
       <div class="ts-banner subject subject-{{ts.subject_key}}">
         <i class="fa"></i>
         <h3>{{ts.title}}</h3>
-      </div> 
+      </div>
+      
       <div class="ts-actions">
+        <!-- Display's Teacher set available copies, totatl copies and status 'new' holds count  -->
```

Comment:
How about 
Displays the number of available copies, total number of copies in stock, and number of recent not-yet-processed holds.